x-pack/plugins/infra/tsconfig.json

```diff
@@ -69,7 +69,8 @@
     "@kbn/logs-shared-plugin",
     "@kbn/licensing-plugin",
     "@kbn/aiops-utils",
-    "@kbn/lens-embeddable-utils"
+    "@kbn/lens-embeddable-utils",
+    "@kbn/expressions-plugin"
```

Comment:
This one was probably added by some automation

Reply:
we're now depending on it: [x-pack/plugins/infra/public/components/asset_details/tabs/overview/metrics/chart_utils.ts](https://github.com/elastic/kibana/pull/166078/files#diff-925253843561c714732fbe58a4c3ebdb4940175cb3ded69e22879180c61132cbR10)